libstorage/src/Volume.cc

```diff
@@ -2007,19 +2006,19 @@ int Volume::doCryptsetup()
 		replaceAltName( "/dev/dm-", Dm::dmDeviceName(mnr) );
 		}
 	    else
-	    {
+	        {
 		getMajorMinor( dmcrypt_dev, dummy, minor );
 		replaceAltName("/dev/dm-", Dm::dmDeviceName(minor));
-	    }
+	        }
 
 	    ProcPart p;
 	    unsigned long long sz;
 	    if( p.getSize( Dm::dmDeviceName(minor), sz ))
 		setSize( sz );
 	    }
-	}
+        }
     else
-	{
+        {
```

Comment:
This indeed fix the indentation according to the rest of the code. But I couldn't let pass the opportunity to say this: what an awful indentation! :wink: